apps/experiments/tasks.py

```diff
@@ -153,3 +154,18 @@ def _convert_prompt_builder_history(messages_history):
         elif message["author"] == "Assistant":
             history.append(AIMessage(content=message["message"]))
     return history
+
+
+@shared_task(bind=True, base=TaskbadgerTask)
+def migrate_experiment_to_pipeline_task(self, experiment_id: int) -> dict:
+    """
+    Migrate a single assistant or llm experiment to pipeline experiment.
+    """
+    try:
+        experiment = Experiment.objects.get(id=experiment_id)
+        call_command("migrate_nonpipeline_to_pipeline_experiments", experiment_id=experiment_id, skip_confirmation=True)
```

Comment:
A preferable alternative to calling the command would be to extract the logic into functions which can be used here and in the command. That also makes it easier to test.